examples/MqttClient/MqttClient.ino

```diff
@@ -243,6 +246,7 @@ void setup() {
 #endif
 
   // MQTT Broker setup
+  // For SSL connection, use port 8883 (your modem has to support it)
   mqtt.setServer(broker, 1883);
```

Comment:
How to use the private key and device certificate for this connection (AWS IoT Core)?

Reply:
This is how it looks for the Wifi secure client:
WiFiClientSecure net;
 PubSubClient client;
net.setCACert(cacert);
    net.setCertificate(client_cert);
    net.setPrivateKey(privkey);
    client.setClient(net);